jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt

```diff
@@ -448,7 +448,6 @@ class CodeWhispererService {
     }
 
     private fun checkRecommendationsValidity(states: InvocationContext, showHint: Boolean): Boolean {
-        val userInput = states.recommendationContext.userInputSinceInvocation
```

Comment:
revert?

Reply:
This is actually an unused variable but can revert for the sake of this PR

Reply:
can leave it in, wasn't sure if intentional